src/config/validate.ts

```diff
@@ -101,6 +102,13 @@ function validateUrlLifetime(urlLifetime: string): void {
 	}
 }
 
+function validateUrlExpireFrom(urlExpire: string): void {
+	logger.debug(`Start validateUrlExpireFrom with ${urlExpire}`)
+	if (!urlExpire || (urlExpire != 'create' && urlExpire != 'update')) {
+		throw new InvalidConfigError(`URL_EXPIRE_FROM specified is invalid (received ${urlExpire})`)
```

Comment:
You should probably also mention the available options.
```suggestion
		throw new InvalidConfigError(`URL_EXPIRE_FROM specified is invalid (received ${urlExpire}, expected 'create' or 'update')`)
```